### DISCUSSION TITLE
跨品种标准套利合约代码

### DISCUSSION THREAD

**Discussion by coderleo** [wondertrader/wondertrader#218] [Q&A]
我平时手动在做豆油和棕榈油的标准套利，打算在wtpy上跑量化，期货市场上的豆棕套利合约代码是SPC y2401&p2401,但是在wtpy上无法使用这个代码。我也查过了demo，并没有涉及到跨品种或者跨期套利。想请问，套利的合约代码在wtpy中是怎样的格式或者调用方式

**Comment by InventorHann** [comment]
您好，我是OKX Broker代理

看到您是量化策略方，想和您看下是否有进一步合作的机会

如果您是进行API下单的，可以通过参数的添加，获得额外的返佣比例
